Frontend/src/FacialRecognition/FacialRecognitionSearchModal.jsx

```diff
@@ -104,6 +110,7 @@ const FacialRecognitionSearchModal = ({ onClose, handlePatientClick }) => {
                         )}
                     </div>
                 </div>
+                {error && <p style={{ color: error === "Image Taken Correctly." ? 'green' : 'red' }}>{error}</p>}
```

Comment:
Don't compare your error directly to a string literal; pull out the string into a constant variable and use that to do your comparison